app/models/work.rb

```diff
@@ -0,0 +1,20 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true
+
+  has_many :votes
+  has_many :users, through: :votes
+
+  def self.sort_by_votes
+    # in descending order
+    return self.all.sort_by { |work| -work.votes.count }
+  end
```

Comment:
This works, but it's better to use the database to sort with the `order` method.  You can see [a blog post on the subject](https://tomkadwill.com/2017/12/18/counting-or-sorting-by-an-association-in-a-rails-sql-query.html) to learn more.